newgrf/03151814/global.yaml

```diff
@@ -3,6 +3,3 @@ description: |-
   Replaces default town names with Michigan town names.
   Made by UnitedMi
 url: "https://github.com/UnitedMi/OpenTTD-Michigan-Town-Names"
-tags:
-- "michigan"
```

Comment:
I guess region support for town names is not yet implemented.

Reply:
I am not sure what you base that guess on, but it is wrong. Maybe on this one instance that I missed? I dunno .. plenty of townnames do have a region assigned to them :)

But, it is a manual process of mapping tags to regions, and it is far from perfect. In the end, the authors should be doing this, I am just trying to help them along :)